rest-services/src/main/java/org/kuali/rice/rest/api/actionlist/ActionListRestController.java

```diff
@@ -0,0 +1,212 @@
+                                                                               @ApiParam(value = "Exclude Document Type filter?") @RequestParam(value = "excludeDocumentType", required = false) boolean excludeDocumentType,
+                                                                               @ApiParam(value = "Create Date From filter") @RequestParam(value = "createDateFrom", required = false) Date createDateFrom,
+                                                                               @ApiParam(value = "Create Date To filter") @RequestParam(value = "createDateTo", required = false) Date createDateTo,
+                                                                               @ApiParam(value = "Exclude Create Date filter?") @RequestParam(value = "excludeCreateDate", required = false) boolean excludeCreateDate,
+                                                                               @ApiParam(value = "Last Assigned Date From filter") @RequestParam(value = "lastAssignedDateFrom", required = false) Date lastAssignedDateFrom,
+                                                                               @ApiParam(value = "Last Assigned Date To filter") @RequestParam(value = "lastAssignedDateTo", required = false) Date lastAssignedDateTo,
+                                                                               @ApiParam(value = "Exclude Last Assigned Date filter?") @RequestParam(value = "excludeLastAssignedDate", required = false) boolean excludeLastAssignedDate,
+                                                                               @ApiParam(value = "Delegator Id filter") @RequestParam(value = "delegatorId", required = false) String delegatorId,
+                                                                               @ApiParam(value = "Primary Delegate ID filter") @RequestParam(value = "primaryDelegateId", required = false) String primaryDelegateId,
+                                                                               @ApiParam(value = "Exclude Delegator ID filter?") @RequestParam(value = "excludeDelegatorId", required = false) boolean excludeDelegatorId,
+                                                                               @ApiParam(value = "Primary Delegation Type filter") @RequestParam(value = "delegationType", required = false) String delegationType,
+                                                                               @ApiParam(value = "Exclude Delegation Type filter?") @RequestParam(value = "excludeDelegationType", required = false) boolean excludeDelegationType,
+                                                                               @ApiParam(value = "Sort Criteria") @RequestParam(value = "sortCriterion", required = false) String sortCriterion,
+                                                                               @ApiParam(value = "Sort Direction") @RequestParam(value = "sortDirection", required = false) String dir
+

```

Comment:
All of these properties should be as part of the filter param and not declared individually like this I believe